src/System.Net.Utilities/src/System/Net/NetworkInformation/Ping.Windows.cs

```diff
@@ -0,0 +1,330 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System.ComponentModel;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace System.Net.NetworkInformation
+{
+    partial class Ping
```

Comment:
Looks like this and other instances need explicit visibility (and perhaps should be `sealed`).